src/components/workEntry.js

```diff
@@ -1,24 +1,23 @@
 import React from 'react';
 import '../css/workEntry.css'
 
-function workHistory(props) {
-  const { company, title, image, started, ended, description } = props
-  const startDate = new Date(started)
-  const endDate = new Date(ended)
+function workEntry({entry}) {
```

Comment:
I've moved the name of this function to match the name of the class.

You're passing an `entry`, which is an object - so you need to tell the function to expect an object.